Strip the OpenMP include (no OMP in WASM); the bare `#pragma omp parallel for`
lines remain as ignored comments when -fopenmp is not passed.
long is 32 bits in wasm32. UniqueKMER packs k-mers of up to 32 bases (64
bits) into its uint64 typedef and builds 64-bit edit-distance masks from 1L,
so give both their real width.

diff --git a/src/common.h b/src/common.h
index 823585e..f8b5ecf 100644
--- a/src/common.h
+++ b/src/common.h
@@ -5,8 +5,8 @@
 
 #define _DEBUG false
 
-typedef long int64;
-typedef unsigned long uint64;
+typedef int64_t int64;
+typedef uint64_t uint64;
 
 typedef int int32;
 typedef unsigned int uint32;
diff --git a/src/editdistance.cpp b/src/editdistance.cpp
index 95d655b..b1b118f 100644
--- a/src/editdistance.cpp
+++ b/src/editdistance.cpp
@@ -31,27 +31,27 @@ template<typename T, typename TVALUE>
 unsigned int edit_distance_bpv(T &cmap, char const *vec, size_t const &vecsize, unsigned int const &tmax, unsigned int const &tlen) {
     int D = tmax * 64 + tlen;
     TVALUE D0, HP, HN, VP, VN;
-    uint64_t top = (1L << (tlen - 1));  // 末尾のvectorに適用
-    uint64_t lmb = (1L << 63);
+    uint64_t top = (1LL << (tlen - 1));  // 末尾のvectorに適用
+    uint64_t lmb = (1LL << 63);
 
     for(size_t i = 0; i <= tmax; ++i) {
         VP[i] = 0;
         VN[i] = 0;
     }
     for(size_t i = 0; i < tmax; ++i) VP[i] = ~0;
-    for(size_t i = 0; i < tlen; ++i) VP[tmax] |= (1L << i);
+    for(size_t i = 0; i < tlen; ++i) VP[tmax] |= (1LL << i);
     for(size_t i = 0; i < vecsize; ++i) {
         TVALUE &PM = cmap[vec[i]];
         for(int r = 0; r <= tmax; ++r) {
             uint64_t X = PM[r];
-            if(r > 0 && (HN[r - 1] & lmb)) X |= 1L;
+            if(r > 0 && (HN[r - 1] & lmb)) X |= 1LL;
             D0[r] = (((X & VP[r]) + VP[r]) ^ VP[r]) | X | VN[r];
             HP[r] = VN[r] | ~(D0[r] | VP[r]);
             HN[r] = D0[r] & VP[r];
-            X = (HP[r] << 1L);
-            if(r == 0 || HP[r - 1] & lmb) X |= 1L;
-            VP[r] = (HN[r] << 1L) | ~(D0[r] | X);
-            if(r > 0 && (HN[r - 1] & lmb)) VP[r] |= 1L;
+            X = (HP[r] << 1LL);
+            if(r == 0 || HP[r - 1] & lmb) X |= 1LL;
+            VP[r] = (HN[r] << 1LL) | ~(D0[r] | X);
+            if(r > 0 && (HN[r - 1] & lmb)) VP[r] |= 1LL;
             VN[r] = D0[r] & X;
         }
         if(HP[tmax] & top) ++D;
@@ -91,9 +91,9 @@ unsigned int edit_distance_map_(char const *a, size_t const asize, char const *b
     unsigned int tmax = (asize - 1) >> 6;
     unsigned int tlen = asize - tmax * 64;
     for(size_t i = 0; i < tmax; ++i) {
-        for(size_t j = 0; j < 64; ++j) cmap[a[i * 64 + j]][i] |= (1L << j);
+        for(size_t j = 0; j < 64; ++j) cmap[a[i * 64 + j]][i] |= (1LL << j);
     }
-    for(size_t i = 0; i < tlen; ++i) cmap[a[tmax * 64 + i]][tmax] |= (1L << i);
+    for(size_t i = 0; i < tlen; ++i) cmap[a[tmax * 64 + i]][tmax] |= (1LL << i);
     return edit_distance_bpv<cmap_v, typename cmap_v::mapped_type>(cmap, b, bsize, tmax, tlen);
 }
 
diff --git a/src/genomes.cpp b/src/genomes.cpp
index 66390e7..a0fa664 100644
--- a/src/genomes.cpp
+++ b/src/genomes.cpp
@@ -5,7 +5,7 @@
 #include <sstream>
 #include <memory.h>
 #include <math.h>
-#include <omp.h>
+
 #include <atomic>
 #include "sequence.h"
 
@@ -99,7 +99,7 @@ void Genomes::filterReferenceGenome() {
     int keylen = min(14, mOptions->kmerKeyLen);
     cerr << "--- Calculate unique key coverage in " << keylen << " bp" << endl;
     
-    size_t flagBufSize = 1L << (2*keylen);
+    size_t flagBufSize = 1LL << (2*keylen);
     uint8* flagBuf = new uint8[flagBufSize];
     memset(flagBuf, 0, flagBufSize * sizeof(uint8));
     int blankBits = 64 - 2*keylen;
